microsoft-edge/devtools-guide-chromium/memory-inspector/memory-inspector-tool.md

```diff
@@ -21,48 +22,176 @@ ms.date: 02/15/2022
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.  -->
-# Inspect a JavaScript ArrayBuffer with the Memory Inspector tool
 
+# Inspect a JavaScript ArrayBuffer using Memory Inspector
+
+Use Memory Inspector to view JavaScript ArrayBuffers and the WebAssembly (WASM) resizable ArrayBuffer `WebAssembly.Memory`. You can open Memory Inspector from the [DevTools menu](#open-memory-inspector-from-the-devtools-menu) or [while debugging](#open-memory-inspector-while-debugging).
+
+<!-- add more scenario type info, ask Zoher -->
```

Comment:
```suggestion
The Sources tool provides a limited ability to inspect `ArrayBuffers` when debugging. The **Scope** view in the Sources tool displays a list of single values within the array buffer, which makes it difficult to see all of the data. Additionally, navigating to a certain range within the buffer requires you to scroll to a specific index and the values are always displayed as a single byte, even if you wanted to see them in another format, like 32-bit integers.
```

This blog post: https://developer.chrome.com/blog/memory-inspector/ was helpful for understanding some of these details. I would also propose adding this image to show the **Scopes** view in the Sources tool and its limitations for array buffers: https://wd.imgix.net/image/dPDCek3EhZgLQPGtEG3y0fTn4v82/SUDWzfUA2n6KopYBHxKh.png?auto=format&w=845

Reply:
added text and new image